gui-wm/hyprland/hyprland-9999.ebuild

```diff
@@ -63,6 +63,20 @@ BDEPEND="
 
 PATCHES=( "${FILESDIR}/${PN}-0.10.3beta-system-wlroots.patch" )
 
+src_prepare() {
+	default
+
+	# Nvidia patch
+	if use video_cards_nvidia; then
+		sed -i "s|glFlush();|glFinish();|" \
+			"${S}/subprojects/wlroots/render/gles2/renderer.c" || die "Nvidia patch failed"
+
+		eapply "${FILESDIR}/0001-Screencast-for-Nvidia.patch"
```

Comment:
Add the file to `PATCHES+=( ${FILESDIR}/0001-Screencast-for-Nvidia.patch )`. Then you don't need to do `eapply` manually.